package.json

```diff
@@ -175,4 +176,4 @@
     "./lib/reanimated2/core",
     "./lib/index"
   ]
-}
+}
```

Comment:
unacceptable 😵 
```suggestion
}

```

Reply:
Sry man fixed in [4419eee](https://github.com/software-mansion/react-native-reanimated/pull/4904/commits/4419eee9b48dfb67c71dc5e62772d1d6f6db8625)